source/_components/noonlight.markdown

```diff
@@ -0,0 +1,129 @@
+---
+layout: page
+title: "Noonlight"
+description: "24/7 Smart Home Monitoring and 9-1-1 integration"
+date: 2019-06-19 12:30
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: noonlight.png
+ha_category:
+  - Switch
+ha_release: 0.96
+---
+
+[Noonlight](https://noonlight.com) connects your smart home to local emergency services to help keep you safe in case of a break-in, fire, or medical emergency.
+
+<p class='note warning'>
```

Comment:
I was under the impression that Noonlight only dispatched security keys to partners, not to public/individual users. Is that changing? Will anyone be able to sign up? If not, we should note _who_ this is available to.

Reply:
That is correct that Noonlight only gives API keys to partners, but Konnected (my company) is a Noonlight partner. We maintain an "authentication broker" web service on AWS that facilitates an OAuth token exchange/renewal for Home Assistant.

Anyone (in the U.S.) will be able to sign up using the link in the associated documentation.

Reply:
@heythisisnate Wow! What a generous move on your part. Nice work!